upstream-community-operators/istio/istio.package.yaml

```diff
@@ -0,0 +1,5 @@
+#! package-manifest: deploy/chart/catalog_resources/rh-operators/istio-operator.0.1.4.clusterserviceversion.yaml
+packageName: istio-operator
```

Comment:
Please make sure the directory name matches the operator's packagename, or vice-versa

Reply:
fixed, thanks @SamiSousa!